File: app/src/main/java/io/github/anvell/stackoverview/model/Owner.java
package io.github.anvell.stackoverview.model;

import com.google.gson.annotations.SerializedName;

public class Owner {

    @SerializedName("profile_image")
    public String profileImage = "";

    @SerializedName("display_name")
    public String displayName = "";

    @SerializedName("link")
    public String link = "";
}

File: app/src/main/java/io/github/anvell/stackoverview/model/Owner.kt
package io.github.anvell.stackoverview.model

import com.google.gson.annotations.SerializedName

data class Owner (
    @SerializedName("profile_image") val profileImage: String,
    @SerializedName("display_name") val displayName: String,
    @SerializedName("link") val link: String
)